src/hacbs/components/PipelineRunDetailsView/tabs/PipelineRunDetailsTab.tsx

```diff
@@ -0,0 +1,117 @@
+import * as React from 'react';
+import {
+  DescriptionList,
+  DescriptionListGroup,
+  DescriptionListTerm,
+  DescriptionListDescription,
+  Title,
+} from '@patternfly/react-core';
+import { calculateDuration } from '../../../utils/pipeline-utils';
+
+type PipelineRunDetailsTabProps = {
+  pipelineRun: any;
+};
+
+const PipelineRunDetailsTab: React.FC<PipelineRunDetailsTabProps> = ({ pipelineRun }) => {
+  const duration = calculateDuration(
+    typeof pipelineRun.status?.startTime === 'string' ? pipelineRun.status?.startTime : '',
+    typeof pipelineRun.status?.completionTime === 'string'
+      ? pipelineRun.status?.completionTime
+      : '',
+  );
+  return (
+    <>
+      <Title headingLevel="h4" className="pf-c-title pf-u-mt-lg pf-u-mb-lg">
+        {' '}
+        Pipelinerun details
+      </Title>
+      <DescriptionList
+        columnModifier={{
+          default: '2Col',
+        }}
+      >
+        <DescriptionListGroup>
+          <DescriptionListTerm>Name</DescriptionListTerm>
+          <DescriptionListDescription>{pipelineRun.metadata.name}</DescriptionListDescription>
+        </DescriptionListGroup>
+        <DescriptionListGroup>
+          <DescriptionListTerm>Status</DescriptionListTerm>
+          <DescriptionListDescription>
+            {pipelineRun.status.conditions[0].status === 'False' ? 'Failed' : 'Succeeded'}
```

Comment:
`pipelineRun.status` is possibly `undefined` here.

Reply:
checked